MAUI/Scheduler/appointments.md

```diff
@@ -1122,6 +1124,77 @@ this.Scheduler.SelectedAppointmentBackground = Brush.Orange;
 
 ![appointment-selection-background-in-maui-scheduler](images/appointments/appointment-selection-background-in-maui-scheduler.png)
 
+## Customize appointment border style
+
+The appointment border style can be customized using the `AppointmentBorderStyle` property, which allows you to modify the `Stroke`, `CornerRadius`, and `StrokeThickness` of appointments in the Scheduler.
+
```

Comment:
Add Notes - for month view appointment display mode

Reply:
updated